cypress/support/commands.js

```diff
@@ -24,4 +24,149 @@ Cypress.Commands.add('load', () => {
+          dagger: 0,
+          primary_isbn10: "0385547684",
+          primary_isbn13: "9780385547680",
+          publisher: "Doubleday",
+          description: "Samuel Sooleymon receives a basketball scholarship to North Carolina Central and determines to bring his family over from a civil war-ravaged South Sudan.",
+          price: "0.00",
+          title: "SOOLEY",
+          author: "John Grisham",
+          contributor: "by John Grisham",
+          contributor_note: "",
+          book_image: "https://storage.googleapis.com/du-prd/books/images/9780385547680.jpg",
+          book_image_width: 342,
+          book_image_height: 500,
+          amazon_product_url: "https://www.amazon.com/dp/0385547684?tag=NYTBSREV-20",
+          age_group: "",
+          book_review_link: "",
+          first_chapter_link: "",
+          sunday_review_link: "",
+          article_chapter_link: "",
+          isbns: [{
+              isbn10: "0385547684",
+              isbn13: "9780385547680"
+            },
+            {
+              isbn10: "0385547714",
+              isbn13: "9780385547710"
+            },
+            {
+              isbn10: "0593459288",
+              isbn13: "9780593459287"
+            }
+          ],
+          buy_links: [{
+              name: "Amazon",
+              url: "https://www.amazon.com/dp/0385547684?tag=NYTBSREV-20"
+            },
+            {
+              name: "Apple Books",
+              url: "https://du-gae-books-dot-nyt-du-prd.appspot.com/buy?title=SOOLEY&author=John+Grisham"
+            },
+            {
+              name: "Barnes and Noble",
+              url: "https://www.anrdoezrs.net/click-7990613-11819508?url=https%3A%2F%2Fwww.barnesandnoble.com%2Fw%2F%3Fean%3D9780385547680"
+            },
+            {
+              name: "Books-A-Million",
+              url: "https://du-gae-books-dot-nyt-du-prd.appspot.com/redirect?url1=https%3A%2F%2Fwww.anrdoezrs.net%2Fclick-7990613-35140%3Furl%3Dhttps%253A%252F%252Fwww.booksamillion.com%252Fp%252FSOOLEY%252FJohn%252BGrisham%252F9780385547680&url2=https%3A%2F%2Fwww.anrdoezrs.net%2Fclick-7990613-35140%3Furl%3Dhttps%253A%252F%252Fwww.booksamillion.com%252Fsearch%253Fquery%253DSOOLEY%252BJohn%252BGrisham"
+            },
+            {
+              name: "Bookshop",
+              url: "https://du-gae-books-dot-nyt-du-prd.appspot.com/redirect?url1=https%3A%2F%2Fbookshop.org%2Fa%2F3546%2F9780385547680&url2=https%3A%2F%2Fbookshop.org%2Fbooks%3Fkeywords%3DSOOLEY"
+            },
+            {
+              name: "IndieBound",
+              url: "https://du-gae-books-dot-nyt-du-prd.appspot.com/redirect?url1=https%3A%2F%2Fwww.indiebound.org%2Fbook%2F9780385547680%3Faff%3DNYT&url2=https%3A%2F%2Fwww.indiebound.org%2Fsearch%2Fbook%3Fkeys%3DSOOLEY%2BJohn%2BGrisham%26aff%3DNYT"
+            }
+          ],
+          book_uri: "nyt://book/010af0b7-2a30-598f-97f5-1ed63c3426e1"
+        }
+      ]}
+    })
+      cy.get('.listItem').eq(0).click()
+  })
```

Comment:
I know this is huge but I'm under the impression that replicating the real data is best practice, since we have cleaner functions that have a job to do. We can get feedback on this tomorrow during our project checkin.